pytorch_lightning/profiler/pytorch.py

```diff
@@ -159,18 +104,36 @@ def __init__(
 
             export_to_chrome: Whether to export the sequence of profiled operators for Chrome.
                 It will generate a ``.json`` file which can be read by Chrome.
+
             path_to_export_trace: Directory path to export ``.json`` traces when using ``export_to_chrome=True``.
                 By default, it will be save where the file being is being run.
+
             row_limit: Limit the number of rows in a table, ``0`` is a special value that
                 removes the limit completely.
+
             sort_by_key: Keys to sort out profiled table.
+
             record_functions: list of profiled functions which will create a context manager on.
                 Any other will be pass through.
+
             local_rank: When running in distributed setting, local_rank is used for each process
                 to write to their own file if `output_fname` is provided.
+
+        Raises:
+            MisconfigurationException:
+                If arg ``sort_by_key`` is not present in ``AVAILABLE_SORT_KEYS``, or
+                if log file is not a ``.txt`` file.
+            ValueError:
+                If you attempt to stop recording an action which was never started.
         """
+        if profiled_functions is not None:
+            # TODO: DeprecationWarning
+            if record_functions is None:
+                record_functions = profiled_functions
+            else:
+                # TODO; raise MisconfigurationException
+                pass
```

Comment:
`profiled_functions` was renamed to `record_functions`. Should we do this?

Reply:
It was to uniformize the API between both 1.8 and before.

Reply:
I get that but we should deprecate then